provisioning_templates/provision/kickstart_default.erb

```diff
@@ -181,31 +181,28 @@ salt-minion
 <%= section_end -%>
 
 <% if @dynamic -%>
-%pre
-<%= snippet_if_exists(template_name + " custom pre") %>
+# Partition table dynamic generation
```

Comment:
lzap, I was testing custom pre and post stuff and I found that I did not get logs from any pre section. I think I know what is happening. when pre executes, the actual filesystems for OS being installed are not created yet so nothing mounted over /mnt/sysimage , pre creates logs under anaconda file systems in /mnt/sysimage/root/*.log , then file systems are created and mounted over it /mnt and we loose the logs.
I think for pre sections, logs should be directed to /tmp [Anaconda Environment] , then in post nochroot section , we can copy them from /tmp to /mnt/sysimage/root so that those are available after reboot in installed OS

Reply:
Thanks a bunch for testing!

Reply:
Finally fixed, thanks.